recipes/nodejs_from_package.rb

```diff
@@ -29,5 +29,8 @@
 end
 
 node['nodejs']['packages'].each do |node_pkg|
-  package node_pkg
+  package node_pkg do
+    action node['nodejs']['package_action'][node_pkg] if node['nodejs']['package_action'][node_pkg]
+    options node['nodejs']['package_options'][node_pkg] if node['nodejs']['package_options'][node_pkg]
+  end
 end
```

Comment:
I think this should be rewritten as the following so that package installs happen faster:
```ruby
package node['nodejs']['packages'] do
  action node['nodejs']['package_action'][node_pkg] if node['nodejs']['package_action'][node_pkg]
  options node['nodejs']['package_options'][node_pkg] if node['nodejs']['package_options'][node_pkg]
end
```

Reply:
There is a separate set of options for each package. So the suggested code doesn't work.

Reply:
Ah you're correct.